src/pages/editProfile/style.css

```diff
@@ -0,0 +1,79 @@
+.editContainer {
+    grid-column: span 2;
+    margin: 15px;
+}
+
+form {
```

Comment:
Make this more specific, otherwise it will affect every form element in the project